src/app/stake/btc/page.tsx

```diff
@@ -1,86 +1,192 @@
+        { delegations: [], pagination: { next_key: "" } }
+      );
+
+      return flattenedData;
     },
-    {
-      name: "Validator 3",
-      btcPk: "key3",
-      totalDelegation: 1500,
-      commission: "4%",
+    retry: (failureCount: number) => {
+      return !isErrorOpen && failureCount <= 3;
     },
-  ];
+  });
+
+  const delegationsLocalStorageKey =
+    getDelegationsLocalStorageKey(publicKeyNoCoord);
+
+  const [delegationsLocalStorage, setDelegationsLocalStorage] = useLocalStorage<
+    Delegation[]
+  >(delegationsLocalStorageKey, []);
+
+  useEffect(() => {
+    if (!delegations?.delegations) {
+      return;
+    }
+
+    const updateDelegationsLocalStorage = async () => {
+      const { areDelegationsDifferent, delegations: newDelegations } =
+        await calculateDelegationsDiff(
+          delegations.delegations,
+          delegationsLocalStorage
+        );
+      if (areDelegationsDifferent) {
+        setDelegationsLocalStorage(newDelegations);
+      }
+    };
+
+    updateDelegationsLocalStorage();
+  }, [delegations, setDelegationsLocalStorage, delegationsLocalStorage]);
+
+  let totalStakedSat = 0;
+
+  if (delegations) {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    totalStakedSat = delegations.delegations
+      .filter((delegation) => delegation?.state === DelegationState.ACTIVE)
+      .reduce(
+        (accumulator: number, item) => accumulator + item?.stakingValueSat,
+        0
+      );
+  }
 
   return (
-    <div className="flex flex-col items-center justify-center h-screen">
-      <h1 className="text-4xl font-bold mb-8">BTC Staking</h1>
-      <Table.Root className="w-full max-w-4xl">
-        <Table.Header>
-          <Table.Row className="bg-gray-200">
-            <Table.ColumnHeaderCell className="px-4 py-2">
-              Name
-            </Table.ColumnHeaderCell>
-            <Table.ColumnHeaderCell className="px-4 py-2">
-              BTC PK
-            </Table.ColumnHeaderCell>
-            <Table.ColumnHeaderCell className="px-4 py-2">
-              Total Delegation
-            </Table.ColumnHeaderCell>
-            <Table.ColumnHeaderCell className="px-4 py-2">
-              Commission
-            </Table.ColumnHeaderCell>
-          </Table.Row>
-        </Table.Header>
-        <Table.Body>
-          {items.map((item) => (
-            <Table.Row
-              key={item.btcPk}
-              className="cursor-pointer hover:bg-gray-100"
-              onClick={() => handleRowClick(item)}
-            >
-              <Table.Cell className="px-4 py-2">{item.name}</Table.Cell>
-              <Table.Cell className="px-4 py-2">{item.btcPk}</Table.Cell>
-              <Table.Cell className="px-4 py-2">
-                {item.totalDelegation}
-              </Table.Cell>
-              <Table.Cell className="px-4 py-2">{item.commission}</Table.Cell>
-            </Table.Row>
-          ))}
-        </Table.Body>
-      </Table.Root>
-    </div>
+    <Theme

```

Comment:
shouldn't this be in the layout?

Reply:
removed it. The main issue was global.css had a parent level black bg, for I was adding themes per page. It's not required anymore